src/test/java/picard/arrays/CreateBafRegressMetricsFileTest.java

```diff
@@ -0,0 +1,59 @@
+package picard.arrays;
+
+import htsjdk.samtools.metrics.MetricsFile;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import picard.PicardException;
+
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class CreateBafRegressMetricsFileTest {
+
+    private static final File TEST_DATA_DIR = new File("testdata/picard/arrays");
+    private static final File TEST_INPUT_FILE = new File(TEST_DATA_DIR, "BafRegress.output.txt");
+    private static final File TEST_BAD_INPUT_FILE = new File(TEST_DATA_DIR, "input.vcf");
+
+    @Test
+    public void testCreateBafRegressMetricsFile() throws IOException {
+        final File outputBase = File.createTempFile("testCreateBafRegressMetricsFile", "");
+        final File output = new File(outputBase.getAbsolutePath() + "." + CreateBafRegressMetricsFile.FILE_EXTENSION);
```

Comment:
Do you need both `outputBase` and `output`? It seems like you could use `outputBase` as `output` and the code would work the same, as the code doesn't look at the file extension, so it doesn't matter what the filename is. Also if you create both then you have to delete both, and `outputBase` is not currently deleted.